app/templates/client/dev/components/todo/todo.js

```diff
@@ -1,51 +1,75 @@
-;(function(angular)
-{
-  "use strict";
-
-  angular
-    .module('myAwesomeApp')
-    .controller('TodoController', ['$log', 'Todo', 'TodoDAO', function($log, Todo, TodoDAO)
-    {
-      var self = this;
-
-      self.todo = new Todo();
-      self.todos = [];
-
-      self.createTodo = function(todo)
-      {
-        var _onSuccess = function(newTodo)
-        {
-          self.todos.push(newTodo);
-          self.todo = new Todo();
-        };
+(function () {
```

Comment:
I think we'd better keep the IIFE starting with a ```;```.

That way we avoid having problems with minifications:

```js

// this will work

;(function()
{
 // some code here
}(window.angular));(function(){}());

```

```js

// this won't work

;(function()
{
 // some code here
})(window.angular)(function(){}());

```